app/controllers/invitations_controller.rb

```diff
@@ -19,15 +20,15 @@ def create
       end
       builder.with(@invitation).build(email)
     end
-
-    if @invitation.valid?
-      flash[:success] = builder.success_message
-      redirect_to @review
-    else
-      flash[:success] = builder.success_message
+   
+    if builder.errors.any?
+      flash.now[:success] = builder.success_message if builder.successes.any?
       flash.now[:alert] = builder.error_message
       @ac = @review.associate_consultant
-      render 'new'
+      render 'new' 
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.<br>Trailing whitespace detected.